mlflow/store/file_store.py

```diff
@@ -59,6 +63,7 @@ def _read_persisted_run_info_dict(run_info_dict):
     dict_copy = run_info_dict.copy()
     if 'lifecycle_stage' not in dict_copy:
         dict_copy['lifecycle_stage'] = LifecycleStage.ACTIVE
+    dict_copy['status'] = RunStatus.to_string(run_info_dict.get('status', RunStatus.RUNNING))
```

Comment:
Maybe add a comment like below indicating why we're making this change. Also do we need a similar isinstance?

Reply:
yes. `FileStore` writes status as enum, where as `SqlAlchemyStore` writes it as a string. This is weird, but the APIs out of these stores are `RunInfo`